app/src/main/res/drawable/ic_home.xml

```diff
@@ -1,3 +1,12 @@
-<vector xmlns:android="http://schemas.android.com/apk/res/android" android:width="24dp" android:height="24dp" android:viewportWidth="24" android:viewportHeight="24">
-    <path android:pathData="M12.786 3.049c0.915 0.975 2.164 1.524 3.487 1.524 0.743 0 1.47-0.177 2.143-0.517 0.256-0.13 0.556 0 0.654 0.263l0.023 0.083 0.007 0.084v7.321c0 2.064-1.059 4.034-3.058 5.874-0.648 0.597-1.37 1.153-2.14 1.66l-0.44 0.28c-0.143 0.087-0.279 0.167-0.408 0.24l-0.187 0.106-0.152 0.08c-0.107 0.056-0.23 0.068-0.345 0.035l-0.083-0.033-0.086-0.046L11.84 19.8c-0.177-0.103-0.36-0.215-0.55-0.335-0.771-0.492-1.502-1.04-2.166-1.633-1.926-1.719-3.026-3.558-3.2-5.487l-0.018-0.263L5.9 11.807V4.541c0-0.351 0.36-0.585 0.674-0.435 0.646 0.308 1.336 0.467 2.04 0.467 1.323 0 2.572-0.55 3.487-1.524 0.187-0.199 0.499-0.199 0.685 0zm-0.343 0.999L12.25 4.217c-1.026 0.85-2.301 1.317-3.636 1.317-0.36 0-0.719-0.035-1.073-0.104L7.276 5.372l-0.322-0.09-0.108-0.036v6.561c0 1.592 0.768 3.162 2.236 4.667l0.237 0.237 0.229 0.216 0.2 0.183c0.625 0.558 1.316 1.076 2.046 1.54l0.234 0.148 0.228 0.136 0.21 0.121-0.015 0.046 0.192-0.107 0.181-0.107 0.19-0.116 0.209-0.13 0.165-0.108c0.729-0.478 1.41-1.003 2.019-1.564 1.63-1.5 2.56-3.068 2.721-4.66l0.02-0.25 0.006-0.252-0.001-6.599-0.051 0.02c-0.303 0.102-0.612 0.18-0.927 0.232l-0.317 0.042-0.32 0.024-0.265 0.007c-1.245 0-2.44-0.406-3.43-1.155l-0.21-0.166-0.19-0.165zm2.523 5.503l0.527 0.538v0.14l-3.42 3.489h-0.142l-2.023-2.063v-0.14l0.528-0.539h0.143L12 12.428l2.822-2.877h0.143z" android:fillColor="?colorPrimary"/>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This doesnt entirely seem to make sense to me - we are selecting for color here, but also using two completely different `VectorDrawable`-assets. What then is the point of using the color selectors? Will they ever even match?

Reply:
ye i forgot to remove this